src/services/timezoneDetectionService.js

```diff
@@ -27,6 +27,13 @@ import jstz from 'jstz'
  * @returns {String} Current timezone of user
  */
 export default () => {
+	if (window.Intl && typeof window.Intl === 'object') {
+		const { timeZone } = Intl.DateTimeFormat().resolvedOptions()
```

Comment:
Not sure if Intl can be defined but not `Intl.DateTimeFormat`. Both seem to have the same percentage on CanIUse.